.github/workflows/android_apk_build.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build Android APK debug file
+
+on:
+  push:
+    paths:
+    - 'android/**'
+    - '.github/workflows/android_apk_build.yaml'
+
+jobs:
+  build:
+    name: Checkout and package
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout to push branch
+        uses: actions/checkout@master
+        with:
+          ref: ${{ github.ref }}                                                              # checkout to branch which was pushed
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Build APK file
+        if: job.status == 'success'
+        run: |
+          cd android
+          wget https://eaterslab.herokuapp.com/api.yaml
+          chmod +x ./api_generate_kotlin.sh
+          ./api_generate_kotlin.sh api.yaml
+          export GRADLE_USER_HOME=$(pwd)/.gradle
+          chmod +x ./gradlew
+          ./gradlew assembleDebug
+      - name: Upload
```

Comment:
What happens if I push multiple commits, will second and other get get rejected because of upstream changes?

Reply:
every commits runs this workflow and the last one finished will be saved as the last workflow result artifact

Reply:
I think there will be conflicts between binaries from different branches coming to master. It would be etter if we keep builds from master in releases and other builds somewhere else, like Heroku or so.

Reply:
will be amnaged when django api for download artifacts will be read, for now I would leave this as simple as possible